src/StackAdmin/AzureRM/AzureRM.psd1

```diff
@@ -51,9 +51,9 @@
     # ProcessorArchitecture = ''
 
     # Modules that must be imported into the global environment prior to importing this module
-    RequiredModules        = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.6.0'; },
+    RequiredModules        = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.7.0'; },
```

Comment:
@bganapa should this match the version of Profile that we have Compute depend on below? (5.8.2)

Reply:
Yes . Updated it.